server/views/components/figure/index.njk

```diff
@@ -18,8 +18,10 @@
       {%- endfor %}"
       alt="{{ caption }}" />
   {% endif %}
-  <figcaption class="figure__caption">
-    {% component 'icon', {icon: icon, extraClasses: ['figure__icon']} %}{% endcomponent %}
-    {{ caption }}
-  </figcaption>
+    {% if full %}<div class="container">{% endif %}
```

Comment:
Surely we want a caption on all images? Unless I am misunderstanding what `full` means?